src/kit/taosdump/taosdump.c

```diff
@@ -483,7 +483,14 @@ static error_t parse_opt(int key, char *arg, struct argp_state *state) {
                 errorPrintReqArg2("taosdump", "P");
                 exit(EXIT_FAILURE);
             }
-            g_args.port = atoi(arg);
+
+            uint64_t port = atoi(arg);
+            if (port > 655325) {
```

Comment:
Should the maximum port number be 65535?

Reply:
good catch!